scss/misc/_delta-buttons.scss

```diff
@@ -29,3 +30,5 @@ button.delta-button-round {
     background-clip: unset;
   }
 }
+
+
```

Comment:
Some not needed newlines :)

Reply:
Alright, I'll remove the lines, thanks for the correction =)